NEWS.md

```diff
@@ -6,9 +6,12 @@ shiny 1.0.3.9001
 ### Breaking changes
 
 ### New features
+* Added an `onStop` argument to `shinyApp()`, which can be used to register a function callback to be run after the app has finished running (all sessions have been disconnected). This is useful if you have finalization/cleaning up code that should be run after the app is closed. If you are creating an app interactively at the R console, this would be the equivalent of code that comes immediately after the `shinyApp()` call. We've also added an `onStop()` function (whose only argument must be the callback function), which has the same effect. This is only necessary for apps that use the two-file format (ui.R and server.R), but it has the same effect regardless of your app structure. In case that you supply both an `onStop()` function _and_ an `onStop` argument to `shinyApp()`, the latter will be the one respected. ([#1770](https://github.com/rstudio/shiny/pull/1770)
```

Comment:
That's true... By "all sessions have been disconnected," I just wanted to make it clear that the scope here is global and not session-level. I felt like "after the app has finished running" part alone could easily be misunderstood as a session ending, especially because people often don't even realize these are different at all. Do you think there's a better way of saying that or we should we just leave the parens out and keep the rest the same?